pkg/cert/utils/utils_certificate_test.go

```diff
@@ -0,0 +1,162 @@
+package utils_test
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"net"
+	"strings"
+
+	api "github.com/gardener/cert-management/pkg/apis/cert/v1alpha1"
+	"github.com/gardener/cert-management/pkg/cert/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("UtilsCertificate", func() {
+	var csr []byte
+	var exampleCn string
+	var exampleSan []string
+	var exampleIPs []net.IP
```

Comment:
Nit: What about using a `var`-block like you did in the previous test?
```suggestion
        var (
        	csr []byte
		exampleCn string
		exampleSan []string
		exampleIPs []net.IP
        )
```